templates/common/infra/terraform/core/host/appservice/appservicenode/appservicenode_output.tf

```diff
@@ -0,0 +1,8 @@
+output "DEFAULT_HOST_NAME" {

```

Comment:
In bicep we output this as URI with the https:// prefix

Reply:
updated